databricks/koalas/frame.py

```diff
@@ -6448,6 +6448,68 @@ def swaplevel(self, i=-2, j=-1, axis=0) -> "DataFrame":
 
         return DataFrame(internal)
 
+    def swapaxes(
+        self, i: Union[str, int] = 0, j: Union[str, int] = 1, copy: bool = True
+    ) -> "DataFrame":
+        """
+        Interchange axes and swap values axes appropriately.
+
+        Parameters
+        ----------
+        i: {0 or 'index', 1 or 'columns'}, default 0. The axis to swap.
+        j: {0 or 'index', 1 or 'columns'}, default 1. The axis to swap.
+
+        Returns
+        -------
+        DataFrame
+
+        .. note:: This method is based on an expensive operation due to the nature
+            of big data. Internally it needs to generate each row for each value, and
+            then group twice - it is a huge operation. To prevent misusage, this method
+            has the 'compute.max_rows' default limit of input length, and raises a ValueError.
+
+                >>> from databricks.koalas.config import option_context
+                >>> with option_context('compute.max_rows', 1000):  # doctest: +NORMALIZE_WHITESPACE
+                ...     ks.DataFrame({'a': range(1001)}).swapaxes()
+                Traceback (most recent call last):
+                  ...
+                ValueError: Current DataFrame has more then the given limit 1000 rows.
+                Please set 'compute.max_rows' by using 'databricks.koalas.config.set_option'
+                to retrieve to retrieve more than 1000 rows. Note that, before changing the
+                'compute.max_rows', this operation is considerably expensive.
```

Comment:
Could you move this to above `Parameters`? This place breaks the doc format:

<img width="754" alt="Screen Shot 2020-12-03 at 4 51 56 PM" src="https://user-images.githubusercontent.com/506656/101107780-e2fcaf80-3587-11eb-89d1-21c82b91a5b2.png">

Reply:
`./dev/lint-python` generates the HTML version, or `cd docs/; make html`?

Reply:
Got it! Thanks!

Reply:
![image](https://user-images.githubusercontent.com/47337188/101111965-2d365e80-3591-11eb-85e3-ffa2bd25933b.png)